### ISSUE TITLE
Server is lagging despite timings showing no reason for lag

### ISSUE THREAD

**Issue by salomj** [aikar/timings#142]
https://timings.aikar.co/?id=4c5c9c8e079a4ce1bdf425b593be3812

I do not see any reason why the server should be lagging. At least not according to this timing report.

**Comment by electronicboy** [comment]
This repo is for the timings viewer and issues with it, for support with actual reports, please speak to the vender of the software you're using

**Comment by salomj** [comment]
And whom would that be?

**Comment by electronicboy** [comment]
according to the report, mohist

**Comment by salomj** [comment]
Well, it would still be nice to have better information on what is causing my server to lag other than just the overview saying `Minecraft::Full Server Tickcount(2751)  total(100.00% 1909.413s, 1,388.16% of tick)`, but all items under that don't even add up to a tenth of that time. Worth nothing, I have switched to Magma and to forge directly, but still get the same level of lag, and none of them indicate further than just that the `Full Server Tick` is lagging. All other items added together don't even hit 40% of a tick, but the full sever tick is more than 1000%.

**Comment by electronicboy** [comment]
The only way to have more info is for stuff to hook into timings and record said info, it's not magical; if magma or mods don't add the relevant hooks for stuff, it's gonna show up under where it's running, if that's in the middle of the tick outside of anything, it's gonna show up in the full server tick as the self time; if you need deeper analysis than what is being provided to timings, you'll need to use tooling which doesn't rely on hooks, i.e. a sampler

**Comment by salomj** [comment]
I have the `https://www.curseforge.com/minecraft/mc-mods/sampler` mod install. I am not sure how to read the results from it. 

It is pretty frustrating though to see my TPS at 5 when no chunks are loaded and no players are in the server. https://timings.aikar.co/?id=e0e2665a22b04487b91d00b3978839b9

Any thoughts I what I can do to discover the source of the issue?

**Comment by salomj** [comment]
The issue was that the pipes from one of the mods was using a method not tracked by this lag tool. I forget which modpack it was, but I removed all the pipes and items from the modpacks Mekanism (excluding gas pipes, as no other mod has gas pipes), Refined Storage, and Thermal Foundations. And then replaced them with Applied Energistics 2 and EnderIO, which were significantly less laggy and was able to get reasonable tps, especially when it wasn't under load. I also significantly broke up my AE2 computer, which also seemed to help reduce lag. I also removed all Mekanism machines from automatic output/input, as EnderIO and AE2 is much less laggy than Mekanism's automatic IO.
